app.py

```diff
@@ -1,12 +1,33 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+from session_items import get_items, get_item, add_item, save_item
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return renderIndex()
+
+@app.route('/listitem', methods=['POST'])
+def addListItem():
+    print("Adding Item!")
+    print('Title=' + request.form.get('textbox'))
+    add_item(request.form.get('textbox'))
+    return renderIndex()
```

Comment:
You'll want to redirect to keep the searchbar url consistent

Reply:
aha thanks